### PR TITLE
Test funnel utils

### PR DESCRIPTION
**Pull request by imagineLife**
_increase code coverage on a few files_

## Description
- [x] test `FunnelUtils`
- [x] increase coverage of `DefaultTooltipContent` 

## Related Issue

https://github.com/recharts/recharts/issues/3976

## Motivation and Context
_increasing code-coverage_

## Screenshots (if appropriate):
Screenshot 2023-11-19 at 10 46 26 AM
Screenshot 2023-11-19 at 10 46 37 AM

## Checklist:
- [x] My code follows the code style of this project.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: test/component/DefaultTooltipContent.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        payload: { x: 100, y: 200, z: 200 },
+        dataKey: 'y',
+        type: 'none',
+      },
+    ],
+    itemSorter: (d: { name: string }) => d.name,
+  };
+  it('covers "itemSorter" prop', () => {
+    // @ts-expect-error ignore payload.type prop for code coverage
+    render(<DefaultTooltipContent {...mockProps} />);
```
Comment (nikolasrieble) [inline test/component/DefaultTooltipContent.spec.tsx:59]: What are we testing here? That the test runs without error? Naively, I would expect an `expect`

File: test/component/DefaultTooltipContent.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        unit: 'kg',
+        value: 200,
+        payload: { x: 100, y: 200, z: 200 },
+        dataKey: 'y',
+        type: 'none',
+      },
+    ],
+    itemSorter: (d: { name: string }) => d.name,
+  };
+  it('covers "itemSorter" prop', () => {
```
Comment (nikolasrieble) [inline test/component/DefaultTooltipContent.spec.tsx:57]: Ideally, test names describe use cases as opposed to technical details i.e. `covers abc`

File: test/component/DefaultTooltipContent.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    // @ts-expect-error ignore payload.type prop for code coverage
+    render(<DefaultTooltipContent {...mockProps} />);
+  });
+  it('covers "labelFormatter" prop', () => {
+    // @ts-expect-error ignore payload.type prop for code coverage
```
Comment (nikolasrieble) [inline test/component/DefaultTooltipContent.spec.tsx:62]: What is the issue with payload.type? What is the type expected here? Can we fix this?

**Review by nikolasrieble — COMMENTED** [review]
Thank you for increasing code coverage. I have a few questions.

**Comment by imagineLife** [comment]
hi @nikolasrieble, thanks for the feedback - I renamed the test & adjusted the props to remove the ts-lint err ✅

**Comment by nikolasrieble** [comment]
@imagineLife Great! Thank you. Only one question remains then: https://github.com/recharts/recharts/pull/3977/files#r1404873088

File: test/component/DefaultTooltipContent.spec.tsx (hunk 1 above)
Comment (imagineLife) [reply test/component/DefaultTooltipContent.spec.tsx]: Hey @nikolasrieble I updated the test again - this test is a "smoke" test & includes an `expect` statement ✅ 🙏

**Review by nikolasrieble — APPROVED** [review]
🚀

**Pull request merged**